templates/archives/stories/women_in_symbols-teaser.jinja2

```diff
@@ -0,0 +1,9 @@
+{% extends 'archives/story_card.jinja2' %}
+
+{% block story_title %}
+Women in Symbols
+{% endblock %}
+
+{% block story_teaser %}
+Although it is very difficult to find material evidence of women’s involvement in computer science, some internal documents from the MIT Computation Center shed light on some of their work.
```

Comment:
Is it true that it is difficult to find material evidence of women's involvement in CS? We have a number of books detailing this history in the lab, and read a *NYT* article together about this. I would temper this argument a bit to explain that not all of women's contributions are obvious, which is what you have done in your write-up.